src/NServiceBus.Core/OpenTelemetry/Metrics/Meters.cs

```diff
@@ -16,4 +16,13 @@ class Meters
 
     internal static readonly Counter<long> TotalFailures =
         NServiceBusMeter.CreateCounter<long>("nservicebus.messaging.failures", description: "Total number of messages processed unsuccessfully by the endpoint.");
+
+    internal static readonly Counter<long> TotalImmediateRetries =
```

Comment:
Why not move these meters into the recoverability folder?

Reply:
We might (or might not ;-) address it in subsequent PRs that Mauro is working on.